topmodel/create_artifacts.jl

```diff
@@ -0,0 +1,53 @@
+# The following `nc_path` is missing intentionally - it must be replaced with paths to the nc file
+# on your local machine. The file is 12GB in size.
+
+using NCDatasets
+using Statistics
+using ClimaArtifactsHelper
+import OrderedCollections: OrderedDict
+
+nc_path = nothing
```

Comment:
We can avoid this by making the script takes in an argument when you run it, so the user would call `julia --project=. create_artifacts.jl name_of_nc_path`.

Reply:
good idea!